drf_to_mkdoc/utils/commons/path_utils.py

```
@@ -0,0 +1,76 @@
+            function = import_string(func_path)
+        except ImportError:
+            logger.warning("PATH_PARAM_SUBSTITUTE_FUNCTION is not a valid import path")
+
+    # If custom function exists and returns a valid value, use it
+    PATH_PARAM_SUBSTITUTE_MAPPING = drf_to_mkdoc_settings.PATH_PARAM_SUBSTITUTE_MAPPING
+    if callable(function):
+        try:
+            result = function(path, parameters)
+            if result and isinstance(result, dict):
+                PATH_PARAM_SUBSTITUTE_MAPPING.update(result)
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)